### PR TITLE
WIP: Unique approach

### PR DESCRIPTION
**Pull request by adamduren**
@zacharytamas Adding you to this so you can see what I'm working to / plan on working on.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="../build/styles/core.css" inline>
   <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="lib/font-awesome/css/font-awesome.min.css">
```
Comment (zacharytamas) [inline src/index.html]: How heavy is this?

File: src/index.html [hunk 2]
```diff
@@ -31,5 +32,16 @@ <h1 class="lead">Apps to <em>empower</em> your business</h1>
 
   </div>
 
+  <div class="unique-approach"></div>
+
+  <footer class="mastfoot">
```
Comment (zacharytamas) [inline src/index.html]: "Mastfoot" is not really a thing. :P

File: src/styles/core.scss [hunk 3]
```diff
@@ -7,12 +7,15 @@
...
+  font-family: Helvetica, Arial, Sans-Serif;
   margin: 0;
 }
 
-header.masthead {
+.masthead {
   height: 635px;
-  background: #3f5e6e;
+  color: white;
+  background: url(../assets/img/header.jpg) no-repeat top left;
```
Comment (zacharytamas) [inline src/styles/core.scss]: I was going to use [`background-size: cover`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size) here.

**Comment by adamduren** [comment]
We'll need to do some fine tuning later but I think we can move on to other things if you agree so that you don't have to branch off my branch.

**Pull request merged**